plugins/woocommerce-admin/client/marketing/components/PluginCardBody/PluginCardBody.scss

```diff
@@ -0,0 +1,43 @@
+.woocommerce_marketing_plugin_card_body {
```

Comment:
Wondering if we can use some variables for the gaps and icon size 👌

Reply:
+1. for reference: https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/internal-style-build/abstracts/_variables.scss#L19-L25

Reply:
Done in [`401ef11` (#34415)](https://github.com/woocommerce/woocommerce/pull/34415/commits/401ef118301e0671a12286142021ce9cdc98dd7b).